var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -31,6 +31,10 @@ class Meme(AutotoolsPackage):
     depends_on('mpi', when='+mpi')
     depends_on('imagemagick', type=('build', 'run'), when='+image-magick')
     depends_on('perl-xml-parser', type=('build', 'run'))
+    depends_on('libxml2', type=('build', 'run'))
+    depends_on('libxslt', type=('build', 'run'))
+
+    patch('arm.patch', when='%arm')
```

Comment:
I forgot to ask earlier, but is this a patch you wrote yourself or is it derived from another source?

Reply:
I wrote it myself.